commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/DefaultCommerceMediaResolver.java

```diff
@@ -151,10 +152,12 @@ public String getUrl(
 				return _html.escape(sb.toString());
 			}
 
-			long groupId = GetterUtil.getLong(
-				httpSession.getAttribute(WebKeys.VISITED_GROUP_ID_RECENT));
+			long companyId = GetterUtil.getLong(
```

Comment:
@ethanbustad I am not sure of this, please can yo double check

Reply:
hey @marco-leo I took a look through the changes, and it all looks pretty decent to me.... but I don't think I've seen us use virtual-instance-scoped configuration before, so I can't really speak to that :/. My only concern from what I see is changing the expected param (here [1]) from `groupId` to `companyId`, which I think will require changes elsewhere in the system to match up -- i.e. we'll need to pass the companyId into the URL in certain cases.

[1] https://github.com/liferay/com-liferay-commerce/pull/980/commits/e4fdf52d8698d8aa6a22c04e9d4c362ce3bcfeb5#diff-c2c5fafe90cf2af378911679a035b8dfR209